### PR TITLE
[MM-21974] improve CSS for client certificate screens

### PR DESCRIPTION
**Pull request by Willyfrog**
**Summary**
Current CSS for selecting the certificate and showing the info is not looking good. this attempts to fix it.

**Issue link**

[MM-21974](https://mattermost.atlassian.net/browse/MM-21974)

### CODE REVIEW

**Comment by Willyfrog** [comment]
Here some screenshots on how does it look currently 
@andrewbrown00 let me know if I can help you reviewing the UI somehow.

Screenshot 2020-01-23 at 14 53 57
Screenshot 2020-01-23 at 14 54 02
Screenshot 2020-01-23 at 14 54 07

File: src/browser/components/SelectCertificateModal.jsx [hunk 1]
```diff
@@ -51,23 +51,19 @@ export default class SelectCertificateModal extends React.Component {
     const subjectShort = this.maxSize(cert.subject.commonName, CELL_SIZE);
```
Comment (deanwhillier) [inline src/browser/components/SelectCertificateModal.jsx]: What about using css to truncate the text and add the ellipsis instead? Calculating text width using a variable-width typeface can be unreliable.

**Comment by Willyfrog** [comment]
Updating  screenshots
Screenshot 2020-01-28 at 09 53 22
Screenshot 2020-01-28 at 09 53 28

File: src/browser/components/showCertificateModal.jsx [hunk 2]
```diff
@@ -55,41 +66,49 @@ export default class ShowCertificateModal extends React.Component {
     const creation = utcSeconds(this.state.certificate.validStart);
     const dateDisplayOptions = {dateStyle: 'full', timeStyle: 'full'};
     const dateLocale = 'en-US';
+    console.log(this.state.certificate);
```
Comment (deanwhillier) [inline src/browser/components/showCertificateModal.jsx]: Whoops, I might have left this in ... can we remove please?

File: src/browser/components/showCertificateModal.jsx [hunk 3]
```diff
@@ -24,20 +24,31 @@ export default class ShowCertificateModal extends React.Component {
...
+      return (
+        <Fragment>
+          <dt className={'certificate-key'}>{descriptor}</dt>
+          <dd className={'certificate-section'}><span/></dd>
+        </Fragment>
+      );
+    };
     const certificateItem = (descriptor, value) => {
-      const ddclass = value ? '' : 'emtpyDescriptor';
+      const ddclass = value ? 'certificate-value' : 'emtpy-descriptor';
```
Comment (devinbinnie) [inline src/browser/components/showCertificateModal.jsx]: Non-blocking typo
```suggestion
      const ddclass = value ? 'certificate-value' : 'empty-descriptor';
```

**Comment by esethna** [comment]
I did the UX review on this, we can merge

**Comment by Willyfrog** [comment]
mannually cherry-picked

**Comment by andrewbrown00** [comment]
> I did the UX review on this, we can merge

Thanks @esethna, much appreciated!

**Pull request merged**